commands/command_migrate.go

```diff
@@ -297,6 +297,8 @@ func init() {
 	importCmd.Flags().BoolVar(&migrateNoRewrite, "no-rewrite", false, "Add new history without rewriting previous")
 	importCmd.Flags().StringVarP(&migrateCommitMessage, "message", "m", "", "With --no-rewrite, an optional commit message")
 
+	exportCmd := NewCommand("export", migrateExportCommand)
```

Comment:
I think that it might make sense to bring in a few of the flags from `import`, namely: `--verbose` and `--object-map`. I think that we can leave `--no-rewrite` off for now, but it might make sense to bring that in later.